lib/components/Input/TextInputs.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import classnames from 'classnames'
+import { omit } from 'lodash'
+import { isBrowser } from 'react-device-detect'
+
+import { DEFAULT_INPUT_CLASSES } from '../../constants'
+
+const sanitizeProps = (props) => {
+  return omit(props, [
```

Comment:
nit: We could just do `{ alignLeft, the rest of them, ...sanitizedProps } = props ` and then remove `lodash` here

Reply:
I don't know how this would work as we're looking to remove a bunch of keys from an object, but also do it in such a way that all of the functions below end up with the same props (hence the function `sanitizeProps()` that can be used by multiple other functions.

I would need to see more code of what you're thinking for this.